### ISSUE TITLE
Master build fails due to POSSIBLE DNS SPOOFING DETECTED/REMOTE HOST IDENTIFICATION HAS CHANGED

### ISSUE THREAD

**Issue by HannesWell** [eclipse-m2e/m2e-core#297]
The current `master` build ends with the following warning, which seems to change its state to `failed`:

```
Deploy m2e repo to /home/data/httpd/download.eclipse.org/technology/m2e/snapshots/1.18.2/latest
+ ssh genie.m2e@build.eclipse.org '								rm -rf  /home/data/httpd/download.eclipse.org/technology/m2e/snapshots/1.18.2/latest/* && 								mkdir -p /home/data/httpd/download.eclipse.org/technology/m2e/snapshots/1.18.2/latest'
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@       WARNING: POSSIBLE DNS SPOOFING DETECTED!          @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
The RSA host key for build.eclipse.org has changed,
and the key for the corresponding IP address 198.41.30.203
is unknown. This could either mean that
DNS SPOOFING is happening or the IP address for the host
and its host key have changed at the same time.
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@    WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!     @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
IT IS POSSIBLE THAT SOMEONE IS DOING SOMETHING NASTY!
Someone could be eavesdropping on you right now (man-in-the-middle attack)!
It is also possible that a host key has just been changed.
The fingerprint for the RSA key sent by the remote host is
SHA256:nGZYqqdg4fIcDOwGTJaHuWrqFY+Ti0pU4SqL9gwejKU.
Please contact your system administrator.
Add correct host key in /home/jenkins/.ssh/known_hosts to get rid of this message.
Offending RSA key in /home/jenkins/.ssh/known_hosts:2
RSA host key for build.eclipse.org has changed and you have requested strict checking.
Host key verification failed.
```

Is this something we should be concerned about?

**Comment by mickaelistria** [comment]
cc @eclipsewebmaster: can you please help?

**Comment by mickaelistria** [comment]
See https://bugs.eclipse.org/bugs/show_bug.cgi?id=575403

**Comment by HannesWell** [comment]
Thank you, Mickael!

Fix by @mickaelistria with commit bea454174598e33ded00dad4ec90ea26f6f8388b

The build looks now as before.
